packages/components/psammead-radio-schedule/CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-| 2.0.0 | [PR#3377](https://github.com/bbc/psammead/pull/3377) Remove episodeTitle prop. In ProgramCard, generate episodeTitle with moment using startTime. |
+| 2.0.1 | [PR#3408](https://github.com/bbc/psammead/pull/3408) Use detokenizer to construct duration label |
```

Comment:
As we have a new required prop, I think we need to make this a major version bump - if it gets pulled in automatically it's not going to work as intended :)